src/FraudSession.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Drupal\commerce_bluesnap;
+
+use Drupal\Core\TempStore\PrivateTempStoreFactory;
+
+/**
+ * Client for making requests to the Card/Wallet Transactions API.
+ */
+class FraudSession {
+
+  /**
+   * The shared temporary storage for commerce_bluesnap.
+   *
+   * @var \Drupal\Core\TempStore\SharedTempStore
```

Comment:
**CHANGE REQUEST**

This is actually the private user tempstore. Correct description and var.

Reply:
Corrected